### PR TITLE
Fried Hedgehog 07_01_CIEL07_1 TL

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 1]
```diff
@@ -0,0 +1,110 @@
+<Page1>
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:1]: Note that this file should be named `07_01_CIEL07_1.txt`

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+After around ten seconds, Hisui entered the room.
+<Page6>
+"Pardon the intrusion."
+"Morning, Hisui. As always, thanks."
+<Page7>
+"Good morning, Shiki-sama. I see that you are already awake."
+"I slept like a log yesterday, so yeah. I’ll head down to the dining hall after I get changed. I’m feeling really hungry, so could you let Kohaku-san know to make my breakfast portions a bit bigger than usual?"
+<Page8>
+"I understand. I’m sure Nee-san will be delighted."
+Hisui bowed and exited the room.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:22]: > 　翡翠は一礼して退室しようとする。
Hisui bows and exits the room

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+"Yes. Your complexion has improved significantly, Shiki-sama. As your personal caretaker, there is no greater joy in this world than seeing you well."
+<Page10>
+So she said, and promptly exited the room.
+"――――――"
+Perhaps, to Hisui, those words were to be spoken as a natural part of her occupation. However, it felt like a sneak attack to me.
+"Ah, um... Yeah, well, thanks."
+As I scratched my flushing cheeks, I blabbered out something only I could hear.
+<Page11>
+"―――No, now’s not the time for this.
+I’m feeling good in a good while, I need to change quickly."
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:35]: > せっかく体調がいいんだ、てきぱき着替えないと」
It's not every day I feel like this. I'd better change quickly."

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+<Page12>
+I switched my gears and shifted to preparing for my morning.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:37]: Good use of the phrase but drop "shifted". Tense also  needs some love:
> 気を取り直して朝の支度をする。
I switch gears and start preparing for the morning.

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+It's Monday.
+The start of the week. A good time to start from scratch.
+<Page13>
+On today’s breakfast menu was Japanese.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:41]: > 朝食は和食だった。
Today's breakfast was traditional Japanese cuisine.

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+White rice and miso soup#.
+Spinach boiled in soy sauce with a squid base.
+And with triops shrimp as the main dish, it was a meal that went easy on the stomach.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:44]: > そしてかぶとえびの炊き合わせを主菜にした、胃に優しい献立である。

"献立" is menu but I think this is acceptable

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+Before I could even relish my meal, my bowl lay empty before me, side dishes exhausted.
+<Page17>
+"...I’m sorry, I didn’t savor this.
+...Um, could I please have seconds?"
+<Page18>
+"Of course. Please give me a moment, I’ll go refill your bowl."
+With that, she took my bowl and was on her way to the kitchen.
+<Page19>
+And while her back was turned, I spoke up,
+"Ah...Then could you-"
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:64]: > 「あ……それなら、その、」

I might expect this to be something closer to
> While you're in there―

But no matter what, needs cjk dash

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+"? Would you like to request something extra?"
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:65]: Hm? Would you like

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+"Yes, by all means. If it comes down to it, I’ll take just a teensy-weensy bit from Akiha-sama’s portions!"
+<Page21>
+"I feel like I don’t want to be around to find out what happens after that..."
+Perhaps feeling elated because of my rare request for seconds, Kohaku-san disappeared into the kitchen.
+<Page22>
+"......But-"
+I’d feel bad if I said this to her, but strangely enough, I still felt somewhat unsatisfied.
+I could feel my appetite’s presence, and the food was delicious too.
+But with every bite I took, my appetite dissipated.
+Like, 'This doesn’t hit the spot' kinda feeling. I craved something less elegantly seasoned and something more... Raw. Like fat, greasy junk.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:78]: fatty, greasy

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+For example, something like raw meat and entrails.
+I could not help but desire something that would remind me of the vividity of life itself, rather than something refined and delicately cooked.
+<Page24>
+With Hisui seeing me off, I leave the Tohno mansion behind me.
+The time is 7:30 AM.
+I did spend some extra time eating, but in the end, I was right back on track at my usual time to school.
+<Page25>
+Monday, the first day of the week. The front of the station is crowded with office workers and students alike, all on their way to their offices and schools respectively.
+This scenery never changes.
+One small thing that has changed though, in front of a certain departmental store, there was a sign that read ‘Entrance Blocked, Do Not Trespass’.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:89]: department store

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+"...That store...#"
+Senpai and co. probably cleaned it up in secret.
+A departmental store that ran through a vampire nest.
+Ever since the catacombs collapsed and Vlov died, this place became just another ruin.
+<Page27>
+"...Oh. It all really is over."
+Senpai had said her purpose here was to get rid of the vampires.
+That she’ll leave the city once that’s over.
+So, I can’t ever meet her again.
+There won’t be any more lunchtimes with me, her and Arihiko. No one to talk about random things with. No one to liven up the mood.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:100]: oxford comma

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 12]
```diff
@@ -0,0 +1,110 @@
...
+<Page28>
+...To be honest, I have a mountain of regrets.
+Just the thought of Senpai not being there even if I go to school, just makes me want to turn around and go back home.
+Even so, Senpai fought to protect this mundaneness of daily life. To feel so bad about it itself is disrespectful to what she did.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:104]: the [mundanity](https://www.merriam-webster.com/thesaurus/mundanity) of daily life

File: script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1 [hunk 13]
```diff
@@ -0,0 +1,110 @@
...
+<Page29>
+Besides, I still remember her.
+'I’ll erase your memories of me tomorrow.', she had said. And yet, she didn’t go through with it.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Ciel/Day 7/07_01_CIEL07_1:107]: > “明日になったら記憶を消す”なんて言っておきながら先輩はそうしなかった。
'Come tomorrow, your memories will be wiped', she said. And yet, she didn’t go through with it.

Despite the phrasing, I don't think this is a direct quote. The closest thing she said was in day 5:
> ですので、貴方の安全を考慮し、ここ一日分の貴方の記憶を消します。いいから抵抗しないように」

**Review by yk1512 — APPROVED** [review]

**Review by Avuxo — APPROVED** [review]

**Pull request merged**